book/book/css/print-9e4910d8.css

```
@@ -0,0 +1,50 @@
+
+#mdbook-sidebar,
+#mdbook-menu-bar,
+.nav-chapters,
+.mobile-nav-chapters {
+    display: none;
+}
+
+#mdbook-page-wrapper.page-wrapper {
+    transform: none !important;
+    margin-inline-start: 0px;
```

Stylelint (17.2.0)

[error] Unexpected unit (length-zero-no-unit)

(length-zero-no-unit)